<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Sadovskiy B. S.</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>601</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="MyTableWidget" name="tableWidgetDagAndDropCustom">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>9</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>10</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>A</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>B</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>C</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>D</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>help</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>Hello</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>World</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>34</string>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string>6</string>
        </property>
       </item>
      </widget>
      <widget class="QTableWidget" name="tableWidgetDragAndDropByDefault">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>5</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>6</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>7</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>8</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>A</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>B</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>Hello</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>World</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>My</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>New</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>One</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelMIMETypeCustomTable">
        <property name="text">
         <string>MIME type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxMimeTypeCustomTable"/>
      </item>
      <item>
       <widget class="QLabel" name="labelColumnDelimiterCustomTable">
        <property name="text">
         <string>Column delimiter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxColumnDelimiterCustomTable">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Comma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Semicolon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tab</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelRowDelimiterCustomTable">
        <property name="text">
         <string>Row delimiter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxRowDelimiterCustomTable">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>CRLF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Line Feed (LF)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Carriage Return (CR)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxQuotesMark">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Quotes mark</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableWidget</class>
   <extends>QTableWidget</extends>
   <header>mytablewidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
